src/test/shared/defaultAwsClientBuilder.test.ts

```diff
@@ -17,29 +17,12 @@ describe('DefaultAwsClientBuilder', () => {
             }
         }
 
-        // We don't want to test against the versions reported by package.json and vscode.version--this is
-        // what the product code does, so these tests would still pass with bad versions. Instead, we
-        // verify that a valid semver is used. This protects us against things like `null`, `undefined`,
-        // or other unexpected values.
-        const semverRegex = (require('semver-regex') as () => RegExp)()
```

Comment:
Can `semver-regex` be removed from package.json?

Reply:
Yep it can, also re-added the test with updated stuff